### ISSUE TITLE
Phase 2 - Implement Remote participation page

### ISSUE THREAD

**Issue by Saallen** [mozilla/2015.mozillafestival.org#179]
Final copy is here: https://docs.google.com/document/d/1Oic1ZK5Ct7ZY8kkTauVwNAuYiZzFwJv7SrcgUxvtvUM/edit

Design is here: https://redpen.io/ao52fe4cf670e63803

@natalieworth - can you please add any necessary assets to this ticket?

**Comment by hannahkane** [comment]
Right at the beginning of November

**Comment by cassiemc** [comment]
Very curious to see what this content might be. How do you suspect remote participation will work? Discourse makes sense. Will each session be listed independently, or is this page more of a general guidelines thing?

**Comment by Saallen** [comment]
@cassiemc We delivered this content via Discourse last year but as the site is currently down I cannot share . I see this page on the website be a general guidelines page with links to the content or discussions on discourse

**Comment by hannahkane** [comment]
Changing this to be an implementation ticket. Design ticket is #195

**Comment by cassiemc** [comment]
Blocked by #195

**Comment by hannahkane** [comment]
This ticket is no longer blocked, but @mmmavis isn't available to build it. Wondering if anyone else can take this on? @ioana-chiorean @Pomax ?

**Comment by natalieworth** [comment]
@hannahkane Here are the files in google drive. Let me know if there is anything else that I can export or if svg's or different file formats are needed for anything. I'll cc @natalieworth (myself) on this in case anything else design related is necessary for development: https://drive.google.com/open?id=0B20-0lpP23c0VDNyWkc0X2FmQ1U

**Comment by Pomax** [comment]
Working on this in https://github.com/mozilla/2015.mozillafestival.org/pull/324, deployed on http://mozfest-2015-with-remote-page.herokuapp.com/remote but this probably requires @mmmavis giving things a look over because I can't seem to get the styling to match the comp at all...

**Comment by fuzzyfox** [comment]
My blog post about remote participation at the fest, looking for tweaks/feedback. /cc @chadsansing @ldecoursy 

I'll post once this bug is closed ;)

**Not public link:** http://blog.wduyck.me/p/05e74ed9-ad91-4469-928a-ac1475ef57e2/

**Comment by chadsansing** [comment]
Awesome, @fuzzyfox. You killed it. Is the remote page live yet? Should 'web we want' in paragraph 3 link to https://webwewant.org/? Since the last sub-header uses 'more,' should the l10n CTA use a different verb, e.g., 'find out how to help,' or, 'find out how to contribute?'

Love the post. So excited!

**Comment by ldecoursy** [comment]
Looking good - I need to take a deeper look after a call, but in the meantime, can we consolidate and just use #mozfestlive as the key hashtag for remote work? @kgorr will also be pre-scheduling some questions to try to engage people throughout the weekend, and will be RTing from @mozteach too.

**Comment by hannahkane** [comment]
@chadsansing - can you please make a clean, bulleted list of any changes you want made to [what's on staging now](http://mozfest-2015-with-remote-page.herokuapp.com/remote)?

**Comment by chadsansing** [comment]
Looks fantastic.

- Pending @fuzzyfox's response to @ldecoursy, we may add #MozFestRemote to the second item in the Conversations list.
- 'How Can MozFest Help Me?' icon looks great, but is awfully close to copy. Increase the whitespace?
- L10n campaign link under projects can now be https://blog.webmaker.org/help-us-get-local-with-web-literacy.
- Link @test @ the Fest, #mozfestlive, and @mozfesttimeline in the 'How Can MozFest Help Me?' paragraph.
- Link Thimble in the 'Work Local. Share Global.' paragraph.

**Comment by hannahkane** [comment]
Thanks, Chad!

@Pomax - please see @chadsansing's comment directly above, and let me know if you'd prefer I file a follow-up ticket for that list.

**Comment by fuzzyfox** [comment]
@ldecoursy We can indeed, was just thinking using something different would prevent the twitter chat downing out the remotee CTAs. If having them merge to one makes tracking easier for you guys then all is cool.

**Comment by Pomax** [comment]
in terms of copy, if someone can drop in the update text then those are easy to add in. @hannahkane the styling needs to be adjusted as part of this PR rather than as followup; we tried reusing style rules we already had, but they seem to be too different to work, so some new rules need to be invented.

**Comment by ldecoursy** [comment]
Rest of the blog looks good to me, but yes, since the tweetchat is also for remotee's, I think it lessens confusion for all to stick with one hashtag (and vote for #mozfestlive)
